test/lib/ufe/testPointInstances.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env mayapy
+#
+# Copyright 2020 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import mayaUtils
+import usdUtils
+
+from mayaUsd import ufe as mayaUsdUfe
+
+from pxr import UsdGeom
+
+import ufe
+
+import unittest
+
+
+class PointInstancesTestCase(unittest.TestCase):
+    '''
+    Tests that the UFE path and scene item interfaces work as expected when
+    working with USD point instances.
+    '''
+
+    _pluginsLoaded = False
+
+    @classmethod
+    def setUpClass(cls):
+        if not cls._pluginsLoaded:
+            cls._pluginsLoaded = mayaUtils.isMayaUsdPluginLoaded()
+
+    def setUp(self):
+        self.assertTrue(self._pluginsLoaded)
+
+        mayaUtils.openPointInstancesGrid14Scene()
+
+    def testPointInstances(self):
+        # Create a UFE path to a PointInstancer prim with an instanceIndex on
+        # the end. This path uniquely identifies a specific point instance.
+        ufePath = ufe.Path([
+            mayaUtils.createUfePathSegment('|UsdProxy|UsdProxyShape'),
+            usdUtils.createUfePathSegment('/PointInstancerGrid/PointInstancer/7')])
+        ufeItem = ufe.Hierarchy.createItem(ufePath)
+
+        # Verify that we turned the path into a scene item correctly and that
+        # we can access the PointInstancer prim.
+        self.assertEqual(ufeItem.nodeType(), 'PointInstancer')
```

Comment:
So this means that the nodeType() of PointInstancer/7 and PointInstancer are the same?  Is that awkward, or is that exactly what we want?

Reply:
Given your other explanations, this makes sense.